giocomo_lab_to_nwb/ophys/giocomoconverter.py

```diff
@@ -21,21 +21,31 @@ def __init__(self, source_data):
         source_data : str
             path to the .mat/sbx file
         """
-        source_data = Path(source_data)
-        assert source_data.suffix in ['.mat','.sbx'], 'source_data should be path to .mat/.sbx file'
-        source_data_dict = dict(
-            SbxImagingInterface=dict(file_path=str(source_data))
-        )
-        s2p_folder = source_data.with_suffix('')/'suite2p'
-        if not s2p_folder.exists():
-            warnings.warn('could not find suite2p')
+        if isinstance(source_data,dict):
+            req_keys = ['SbxImagingInterface', 'Suite2pSegmentationInterface', 'GiocomoVRInterface']
```

Comment:
It would be better to use jsonschema.validate

Reply:
yeah, great point, since nwbconverter already has a method to return a schema check #11